Framework/Algorithms/src/PhaseQuadMuon.cpp

```diff
@@ -92,7 +92,36 @@ std::map<std::string, std::string> PhaseQuadMuon::validateInputs() {
   if (tabWS->columnCount() != 3) {
     result["PhaseTable"] = "PhaseTable must have three columns";
   }
-
+  auto names = tabWS->getColumnNames();
+  for (int j = 0; j < 3; j++) {
+    std::transform(names[j].begin(), names[j].end(), names[j].begin(),
+                   ::tolower);
+  }
+  std::vector<std::string> goodNames = {"phase", "asym", "asymm", "asymmetry"};
```

Comment:
I think it would be good to define these names in an anonymous namespace using a `std::array` so that they are defined only once. Perhaps separate the phase name from the asymmetry? We could even just use a prefix for the asymmetry and use [`boost::istarts_with`](http://www.boost.org/doc/libs/1_41_0/doc/html/boost/algorithm/istarts_with.html) to compare case insensitively to avoid having to transform?